src/components/textField/__tests__/index.driver.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import React, {useState} from 'react';
-import {waitFor} from '@testing-library/react-native';
+import {render} from '@testing-library/react-native';
 import View from '../../../components/view';
-import {TextFieldDriver} from '../TextField.driver';
+import {TextFieldDriver} from '../TextField.driver.new';
```

Comment:
Let's remove the old file

Reply:
I think we can't do it yet because we still have other components that depend on the old ones.